src/main/java/com/slickqa/webdriver/finders/OrBy.java

```diff
@@ -0,0 +1,44 @@
+package com.slickqa.webdriver.finders;
+
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebElement;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class OrBy extends By {
+    private List<By> finders;
+
+    public OrBy(By first, By ...others) {
```

Comment:
I wondered why you had the 2 params at first but realized the ... allows for zero parameters so you went to be sure there's at least one